pkg/agent/evictionmanager/plugin/memory/helper.go

```diff
@@ -161,10 +161,10 @@ func (e *EvictionHelper) getWatermarkMetrics(numaID int) (free, total, scaleFact
 	return free, total, scaleFactor, nil
 }
 
-func (e *EvictionHelper) getSystemKswapdStealMetrics() (float64, error) {
+func (e *EvictionHelper) getSystemKswapdStealMetrics() (*metric.MetricData, error) {
 	m, err := e.metaServer.GetNodeMetric(consts.MetricMemKswapdstealSystem)
 	if err != nil {
```

Comment:
if you want to use m.Time, maybe it's better to check whether it's nil here

Reply:
updated